tests/conftest.py

```diff
@@ -82,23 +82,35 @@ def small_scenario(small_infra: Infra, foo_project_id: int, foo_study_id: int) -
     yield Scenario(foo_project_id, foo_study_id, scenario_id, small_infra.id, timetable_id)
 
 
-def _get_or_create_fast_rolling_stock_id():
-    response = requests.post(f"{API_URL}/rolling_stock/", json=json.loads(FAST_ROLLING_STOCK_JSON_PATH.read_text()))
-    if response.status_code == 200:
-        return response.json()["id"]
-    # Rolling stock already exists
-    response = requests.get(f"{API_URL}rolling_stock/?page_size=1000")
-    rolling_stocks = response.json()["results"]
-    return next(
-        iter([rolling_stock["id"] for rolling_stock in rolling_stocks if rolling_stock["name"] == "fast_rolling_stock"])
+def _create_fast_rolling_stocks(
+    names: Iterable[Optional[str]] = [None], metadata: Iterable[Optional[Mapping[str, str]]] = [None]
```

Comment:
List literals as default arguments is risky, and pycharm complains about it. Is there a reason not to use `None` directly?

`Iterable` don't support the `[]` operator, the type should probably be `List` if it's what's expected

Reply:
Indeed it was not clean, I have changed it for `names_and_metadata: Optional[Mapping[str, Mapping[str, str]]]` :smile: